### PR TITLE
added landing page and alert component

### PR DESCRIPTION
**Pull request by shristabaruah**
- Added landing page
- Main-documentation-Introduction page
- Alert Component
landing page
main
alert1
alert2

<--My logo in alert page is not working,I will work on that later-->

### CODE REVIEW

File: components/component.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400&family=Roboto+Slab:wght@300&display=swap');
+@import url("./alert/alert.css");
+@import url("./avatar/avatar.css");
+:root{
+    --main-color:#0000FF;
+    --black:#000000;
+    --white:#ffffff;
+    --ivory-white:#Fdfff5;
+    --grey:rgb(243, 243, 243);
+
```
Comment (NamrataRaikwar2002) [inline components/component.css:10]: you can remove this empty line

File: components/main.css [hunk 2]
```diff
@@ -0,0 +1,259 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500&display=swap');
+:root{
+    font-family: 'Poppins', sans-serif;;
+}
+.logo a{
```
Comment (NamrataRaikwar2002) [inline components/main.css:5]: you can use beautify file for formatting.

File: components/main.css [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+}
+.sidebar-wrapper{
+    width: 20rem;
+    height: 100%;
+    overflow-y: auto;
+    background-color: var(--grey);
+    position: fixed;
+    font-size: 1rem;
+    z-index: 5;
+    box-shadow: rgba(34, 33, 33, 0.35) 2px 5px 15px;
```
Comment (NamrataRaikwar2002) [inline components/main.css:40]: you should take variables for colors.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+}
+body{
+    background-image: linear-gradient(120deg, #fdfbfb 0%, #ebedee 100%);
+    
+}
+
+.nav-bar{
+    display: flex;
+    background-color: rgb(228, 228, 228);
```
Comment (NamrataRaikwar2002) [inline style.css:15]: you can change it where ever you are using direct color.Use variables.

**Review by NamrataRaikwar2002 — COMMENTED** [review]
All looks good, gave some suggestions.

**Review by anjali1102 — APPROVED** [review]
pretty 😀

**Review by NamrataRaikwar2002 — APPROVED** [review]

File: components/main.css (hunk 2 above)
Comment (Jay-Gadhiya) [reply components/main.css:5]: I suggest prettier because beautify sometimes not arrange code properly.

File: components/main.css (hunk 3 above)
Comment (Jay-Gadhiya) [reply components/main.css:40]: you can make 1 css file and add all repeated colors and classes and then import it into project css file.

**Pull request merged**